source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,276 @@
+   :dedent:
+   :start-after: begin text index
+   :end-before: end text index
+
+.. _text-search-go:
+
+Text Search
+-----------
+
+A text search retrieves documents that contain a **term** or a
+**phrase** in the text indexed fields. A term is a sequence of
+characters that excludes whitespace characters. A phrase is a sequence
+of terms with any number of whitespace characters.
+
+To perform a text search, use the ``$text`` evaluation query operator,
+followed by the ``$search`` field in your query filter. The ``$text`` operator
+performs a text search on the text indexed fields. The ``$search`` field
+specifies the text to search in the text indexed fields.
+
+Query filters for text searches use the following format:
+
+.. code-block:: go
+
+   filter := bson.D{{"$text", bson.D{{"$search", "<text to search>"}}}}
+
+.. _term-search-go:
+
+Search by a Term
+~~~~~~~~~~~~~~~~
+
+To search for a term, specify the term as a string in your query filter.
+To search for multiple terms, separate each term with spaces in the string.
+
+.. note::
+
+   When searching for multiple terms, the ``Find()`` function returns
+   documents with at least one of the terms in text indexed fields.
+
+Example
+```````
+
+The following example runs a text search for titles that contain the term "War":
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/textSearch.go
+   :language: go
+   :dedent:
+   :start-after: begin term search
+   :end-before: end term search
+
+After running this example, the output resembles the following:
+
+.. code-block:: none
+   :copyable: false
+
+   [{_id ObjectID("...")} {title Avengers: Infinity War} {year 2018}]
+   [{_id ObjectID("...")} {title Captain America: Civil War} {year 2016}]
+
+Search by a Phrase
+~~~~~~~~~~~~~~~~~~
+
+To search for a phrase, specify the phrase with escaped quotes as a
+string in your query filter. If you don't add escaped quotes around the
+phrase, the ``Find()`` function runs a :ref:`term search <term-search-go>`.
+
+.. tip::
+
+   Escaped quotes are a backslash character followed by a double quote
+   character.
+
+Example
+```````
+
+The following example runs a text search for titles that contain the
+phrase "Infinity War":
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/textSearch.go
+   :language: go
+   :dedent:
+   :start-after: begin phrase search
+   :end-before: end phrase search
+
+After running this example, the output resembles the following:
+
+.. code-block:: none
+   :copyable: false
+
+   [{_id ObjectID("...")} {title Avengers: Infinity War} {year 2018}]
+
+Search with Terms Excluded
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each term you want to exclude from your text search, specify the
+term prefixed with a minus sign as a string in your query filter.
+
+.. important::
+
+   You must search for at least one term if you want to exclude
+   terms from your search. If you don't search for any terms, the
+   ``Find()`` function doesn't return any documents.
```

Comment:
I suggest replacing the note above with something similar to the following: 

Don't exclude all terms in your text search. You must include at least one term to search so that the ``Find()`` function can return documents.

Reply:
I disagree. To me, it reads as though the user must do a search for terms to exclude terms, which is not the same as all terms in the search can't be excluded only terms (OR) one or more terms should be non-excluded terms (or terms without the minus prefix).